projects/packages/backup/.gitattributes

```diff
@@ -1,6 +1,9 @@
 # Files not needed to be distributed.
 .gitattributes   export-ignore
 .github/         export-ignore
+.gitignore       export-ignore
+phpunit.xml.dist export-ignore
+tests/           export-ignore
 
 # Files not needed in the production build.
-/changelog/**    production-exclude
+/changelog/**    production-exclude
```

Comment:
Total nitpick, but would you mind adding the trailing newlines to these files? Look for that icon.

If you don't want to I won't insist, but some future PR touching the file will probably wind up adding it as an unrelated change.

Reply:
Sure! Thanks for pointing that out, Brad! Fixed.